less_users/tests/test_user_profile.py

```diff
@@ -1,9 +1,33 @@
-from less_users.models import Profile
+from less_users.models import Profile, UserChallenge, Log
 
 
-def test_profile_user_create(user):
+def test_create_user_profile(user):
     profile = user.profile
 
-    assert profile in Profile.objects.all()
+    assert Profile.objects.filter(id=profile.id).exists()
     assert user.profile.points == 0
 
+
+def test_change_user_profile(user):
+    profile = user.profile
+    profile.points = 10
+
+    assert user.profile.points == 10
+
+
+def test_change_user_profile_by_user_update_form(user):
+    pass
+
+
+def test_change_user_profile_points_by_clicking_done(user, challenge_2_week):
+    my_profile = Profile.objects.get(user_id=user.id)
+    user_challenge = UserChallenge.objects.create(user=user, challenge=challenge_2_week)
+    user_challenge.check_if_active()
+    points = challenge_2_week.points
+    if user_challenge.get_points() == points and user_challenge.is_active:
```

Comment:
ale dlaczego if? To ty tworzysz warunki wejściowe do testu, nie powinno być żadnych wątpliwości.

Reply:
zastanawiałam się, czy to potrzebuję, nawet to zakomentowałam na chwilę, ale wskrzesiłam... fakt- bez sensu w teście wątpliwości- przecież ma je rozwiewać